lib/updaters/index.js

```diff
@@ -17,8 +17,14 @@ function getUpdaterByFilename (filename) {
   if (PLAIN_TEXT_BUMP_FILES.includes(filename)) {
     return getUpdaterByType('plain-text')
   }
+  if (/pom.xml$/.test(filename)) {
+    return getUpdaterByType('pom')
+  }
+  if (/build.gradle$/.test(filename)) {
+    return getUpdaterByType('gradle')
+  }
```

Comment:
Instead of baking these here, should we use plugin approach so that any project can hook into the lifecycle?

Reply:
Happy for any suggestions on how to do this! The only goal on my side would still be able to use:

```
npx standard-version --some-flags
```

So that for those who are not using node as the primary language it's still a straightforward one-line operation (installing something like `@standard-version/maven-plugin` would not be suitable as it would require a `package.json` to be present; the great thing about this project is that you don't even need `package.json` - although of course it's convenient for node)